services/runagent/frunssh.cpp

```diff
@@ -56,13 +56,17 @@ int main( int argc, char *argv[] )
 #endif
 
     try  {
+#ifndef _CONTAINERIZED
         StringBuffer logname;
         splitFilename(argv[0], NULL, NULL, &logname, NULL);
 
         Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator("frunssh");
         lf->setCreateAliasFile(false);
         lf->setMsgFields(MSGFIELD_prefix);
         lf->beginLogging();
+#else
+        updateContainerizedLogMsgHandlerLogFields(MSGFIELD_prefix);
```

Comment:
it's v. unlikely we're going to want frunssh in containerized world (we don't now).
So I'd leave this file untouched.